src/components/DateRangePickerInput.jsx

```diff
@@ -123,6 +126,22 @@ export default class DateRangePickerInput extends React.Component {
     });
   }
 
+  renderInputIcon(inputIcon) {
+    const {disabled, phrases.focusStartDate, onArrowDown} = this.props;
```

Comment:
curly braces in JS need padding spaces inside them - the linter should be failing here.